src/store/modules/user/actions.ts

```diff
@@ -204,9 +209,14 @@ const actions: ActionTree<UserState, RootState> = {
   /**
    * update current shopify config id
    */
-  async setCurrentShopifyConfig({ commit, dispatch }, shopifyConfig) {
-    commit(types.USER_CURRENT_SHOPIFY_CONFIG_UPDATED, shopifyConfig ? shopifyConfig : {});
-    dispatch('job/clearJobState', null, { root: true });
+  async setCurrentShopifyConfig({ commit, dispatch, state }, payload) {
+    const shopifyConfigs = state.shopifyConfigs ? state.shopifyConfigs : {};
+    let currentShopifyConfig;
+    payload.shopifyConfig.shopifyConfigId ?
```

Comment:
If we are getting shopifyConfig, just set it as is, if not find and set. Same applied to productStore

Reply:
Made the changes, sir, please review.